packages/contract-tests/test/lib/utils/ens.js

```diff
@@ -26,18 +26,22 @@ const methodABI = {
   type: "function"
 };
 const args = ["my.ens.name", 555, { from: "the.other.name" }];
+const ensSettings = {
+  enabled: true,
+  networkId: 1
+};
 
 describe("convertENSNames", () => {
```

Comment:
This is likely too much work for this PR and should be saved for another time, but I'm wondering, would it be possible to change these tests to not use mocks but rather to use `@truffle/deployer` to set up a real ENS instance on a testnet?  (This is how I'm testing the ENS support in `@truffle/encoder` in #3974, and to my mind it seems like a better test than using mocks.)

Reply:
I figured those kinds of tests would be more appropriate in deployer rather than in contract-tests since they don't use a "real ENS instance". But that sounds like it might be good over there.

Reply:
Yeah I mean probably don't do this on this PR, would be better to just get this merged I think, just noting it for later.

My thought is that the tests make sense here, because -- having tested the deployer elsewhere -- here you *use* the deployer to test this.  (Same as I'm doing in `encoder`.)  I don't mean an instance of the deployer *or* ensjs; I mean an instance of the ENS contracts, the ENS registry.  You use the deployer's functionality where it can set up an ENS registry on a testnet.  Then, instead of mocking, you can test using an actual ENS registry that you set up. :)

That seems like a better test to me, but as noted that probably shouldn't actually go in this PR.